src/main/java/edu/ie3/datamodel/io/factory/timeseries/PsdmTimeBasedWeatherValueFactory.java

```diff
@@ -70,10 +75,10 @@ public String getTimeFieldString() {
     Point coordinate = data.getCoordinate();
     java.util.UUID uuid = data.getUUID(UUID);
     ZonedDateTime time = timeUtil.toZonedDateTime(data.getField(TIME));
-    ComparableQuantity<Irradiation> directIrradiation =
-        data.getQuantity(DIRECT_IRRADIATION, StandardUnits.IRRADIATION);
-    ComparableQuantity<Irradiation> diffuseIrradiation =
-        data.getQuantity(DIFFUSE_IRRADIATION, StandardUnits.IRRADIATION);
+    ComparableQuantity<Irradiance> directIrradiance =
+        data.getQuantity(DIRECT_IRRADIANCE, StandardUnits.SOLAR_IRRADIANCE);
+    ComparableQuantity<Irradiance> diffuseIrradiance =
+        data.getQuantity(DIFFUSE_IRRADIANCE, StandardUnits.SOLAR_IRRADIANCE);
```

Comment:
```suggestion
    ComparableQuantity<Irradiance> directIrradiance =
        data.getQuantity(DIRECT_IRRADIANCE, PowerSystemUnits.WATT_PER_SQUAREMETRE);
    ComparableQuantity<Irradiance> diffuseIrradiance =
        data.getQuantity(DIFFUSE_IRRADIANCE, PowerSystemUnits.WATT_PER_SQUAREMETRE);
```

Would make it explicitly here as well.